### PR TITLE
Simplify export syntax

### PR DESCRIPTION
**Pull request by daniarsuprianto**
Based on new feature added by tailwind https://github.com/tailwindlabs/tailwindcss/pull/2474

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,67 +1,53 @@
...
-      'dark-1': '#333',
-      'dark-2': '#666',
-      'dark-3': '#999',
-      'light-1': '#ccc',
-      'light-2': '#ededed',
-      'light-3': '#f7f7f7',
-      white: '#fff',
-      black: '#000',
-      brown: '#53241b'
+      "orange-1": "#de9000",
```
Comment (jpsc) [inline index.js:18]: Why the hate on " ? 😬

**Review by jpsc — APPROVED** [review]
Don't forget to add proper breaking change commit instructions because this is a breaking change :)

File: index.js (hunk 1 above)
Comment (daniarsuprianto) [reply index.js:18]: it has breaking change flag on my first commit :)

**Comment by daniarsuprianto** [comment]
@jpsc it has breaking change flag on my first commit :)

File: index.js (hunk 1 above)
Comment (daniarsuprianto) [reply index.js:18]: ' is for char, " is for word 😄

**Pull request merged**